lldb/test/API/commands/target/basic/Makefile

```diff
@@ -3,4 +3,8 @@
 # C_SOURCES := b.c
 # EXE := b.out
 
+ifndef PIE
+	LDFLAGS := -no-pie
```

Comment:
Setting LD_EXTRAS is the typical way to pass linker flags from here (the same for other files)